app/konnect/configure/runtime-manager/runtime-groups/index.md

```diff
@@ -0,0 +1,140 @@
+you can configure two additional custom runtime groups in the same Konnect
+account to manage runtime instances and their configuration in any groupings
+you want.
+
+_<diagram - do we have a diagram of how runtime groups work somewhere?>_
+
+## Default runtime group
+
+The default runtime group is the foundational group in Konnect. Every
+organization starts with one default group.
+
+This group can be renamed, but it can't be deleted, and its status as the default
+group can't be changed.
+
+_Q: Is there going to be some identifier for the default group besides the name?
+If the name can be changed, then how would a future admin know which group is the default?_
+
+### Application registration in the Dev Portal
+
+When publishing documentation from the Service Hub to the Dev Portal, you can
+publish API specs from any version -- no matter which runtime group the version
+is in. However, application registration is only supported for Service versions
+running in the default runtime group.
+
+So, if you want a Service version to be available for application registration,
+create the version in the *default* runtime group. This means that you can have a
+mix of Service versions running in different groups for different purposes.
+
+For example, you might have:
+
+* The latest version of a Service published to the Dev Portal with the latest
+spec, running in the *default* group with application registration enabled.
+
+* A previous version of the Service published to the Dev Portal,
+running in the *default* group, but with application registration disabled.
+
+* Another previous version of the Service, published to the Dev Portal, and
+running in a *custom* runtime group. This version would only have the documentation
+available, and developers would not have access to application registration for
+this version.
+
+Out of these three scenarios, only the Service version in the first scenario
+would be available for application registration.
+
+## Multiple runtime groups
+{:.badge .enterprise}
+
+In addition to the default runtime group, you can also configure two additional
+custom groups. Splitting runtime instances and configuration into multiple
+runtime groups provides multiple specialized environments in one account.
+
+Some common use cases for using multiple runtime groups include:
+
+* **Environment separation:** Split environments based on their purpose, such as
+development, staging, and production.
+* **Region separation:** Dedicate each runtime group to a region or group of
+regions. Spin up runtime instances in those regions for each runtime group.
+* **Team separation:** Dedicate each runtime group to a different team and share
+resources based on team purpose.
+
+[Set up and manage runtime groups &rarr;](/konnect/configure/runtime-manager/runtime-groups/manage)
+
+## Administration
+
+Access to each runtime group is configurable on a team-by-team basis using
+entity-specific permissions.
```

Comment:
Couple of questions about permissions (I plan to expand this section when working on specific permissions/teams doc, but couldn't find this info):

* When you create users, do they default to the default runtime group?
* What happens to users when a custom group is deleted? Are they left orphaned and with no permissions, are their accounts placed into some specific state, or are they dumped into the default runtime group? Or something else?

Reply:
> When you create users, do they default to the default runtime group?
No. When you "invite" a new user, you have the option of adding them to a team. The user will have access to the roles granted to their team which may or may not contain the default runtime group.

> What happens to users when a custom group is deleted?
Roles are assigned to users, if a custom RG is deleted, nothing happens to the user. Does that answer your question?